.gitignore

```diff
@@ -11,4 +11,10 @@
 bin/
 include/
 lib/
-pyvenv.cfg
\ No newline at end of file
+pyvenv.cfg
+/.idea/*
+# Some local files
```

Comment:
The `.idea` folder is a good addition.
I am unsure about the local files. I would prefer to not include some "random" filenames.
@MrFlamez @xRuffKez What is your opinion?